index.js

```diff
@@ -1,18 +1,44 @@
-/***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+/***** Deliverable 1 *****/ //[DONE]
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
-/***** Deliverable 2 *****/
+/***** Deliverable 2 *****/ //[DONE]
 
+header.style.color = "blue"
 
+/***** Deliverable 3 *****/ //[DONE]
+console.log('This is what the traveler object looks like: ', traveler)
+const profile = document.getElementById("profile")
 
-/***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+const image = profile.getElementsByTagName("img")[0]
+image.src = traveler.photo
+image.alt = traveler.name
 
+const h2 = profile.getElementsByTagName("h2")[0]
+h2.innerHTML = traveler.name
```

Comment:
```suggestion
const h2 = profile.getElementsByTagName("h2")[0]
h2.textContent = traveler.name
```

Reply:
It's better to use `textContent` over `innerHTML`, especially when you're setting data that you know should be plain text. See here to read more about the security implications of `innerHTML`: https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations